dev/customer_testing/run_tests.dart

```diff
@@ -85,18 +81,18 @@ Future<bool> run(List<String> arguments) async {
   final bool help = parsedArguments['help'] as bool;
   final int numberShards = int.tryParse(parsedArguments['shards'] as String);
   final int shardIndex = int.tryParse(parsedArguments['shard-index'] as String);
-  final List<File> files = parsedArguments
-    .rest
-    .expand((String path) => Glob(path).listFileSystemSync(const LocalFileSystem()))
-    .whereType<File>()
-    .where((File file) => !skipTemplate || path.basename(file.path) != 'template.test')
-    .toList();
+  final List<File> files = parsedArguments.rest
+      .expand((String path) => Glob(path).listFileSystemSync(const LocalFileSystem()))
+      .whereType<File>()
+      .where((File file) => !skipTemplate || path.basename(file.path) != 'template.test')
+      .toList();
```

Comment:
was this an intentional change? i can't quite figure out what the change is. is it just formatting?

Reply:
Formatter, no longer in the split up PR